src/lib/Utils.js

```diff
@@ -38,6 +38,8 @@ export const isKovan = () => {
 
 export const toCommmSepratedString = (n) => n.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
 
+export const percentage = (partialValue, totalValue) =>  (100 * partialValue) / totalValue;
```

Comment:
maybe do the big number translation here?

Reply:
no need  
we
are calculating a percentage of normalized numbers